docs/tutorials/how-to-test-a-contract.md

```diff
@@ -80,12 +61,12 @@ describe('Test SmartContract `Demo`', () => {
 })
 ```
 
-A contract public function must be called on the `self` parameter within the `callPub` callback. It is not recommended to test multiple public functions in the `callPub` callack.
+A contract public method must be called on the `self` parameter within the `callPub` callback. It is not recommended to test multiple public method in the `callPub` callack.
 
 
-### Testing the non-public function
+### Testing non-public methods
 
-Non-public functions have a return value. You can test it directly by asserting its return value. No need to call the verify method.
+Non-public methods have a return value. You can test it directly by asserting its return value. No need to call the verify method.
```

Comment:
Non-public methods have a return value, which can be tested directly. You do not need to call the verify method.